tests/unit/utils/validators.spec.ts

```diff
@@ -26,3 +28,54 @@ test.describe('assertAddress', () => {
     ).toBeUndefined()
   })
 })
+
+test.describe('assertAddresses', () => {
+  test('throw when address is invalid', async ({ page }) => {
+    try {
+      assertAddresses(['0x4440a406E00ECfe15a833119D20B848a31c80116', '0x0'])
+    } catch (error) {
+      expect((error as unknown as Error).message).toBe(`0x0 is not an address`)
```

Comment:
idk about playwright but in jest there is this nice function:

https://jestjs.io/docs/expect#tothrowerror

Is there an equivalent in playwirght? (if so i think it looks easier to read)

Reply:
Yeah, I've initially tried that but here instead of registering the throw test fails because of it 😅 So the workaround is to wrap in try catch. I guess it's because I use Playwright which is more for E2E and not really for unit testing like Jest. But it does the job for now so I didn't want to add Jest yet.